_maps/RandomRuins/IceRuins/icemoon_surface_lust.dmm

```diff
@@ -0,0 +1,171 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/closed/mineral/random/snow/no_caves,
+/area/icemoon/surface/outdoors)
+"b" = (
+/obj/structure/sign/poster/contraband/lusty_xenomorph,
+/turf/closed/indestructible/syndicate,
+/area/icemoon/surface/outdoors)
+"c" = (
+/turf/open/floor/mineral/diamond,
+/area/icemoon/surface/outdoors)
+"d" = (
+/obj/item/reagent_containers/food/drinks/trophy/gold_cup,
+/turf/open/floor/mineral/diamond,
+/area/icemoon/surface/outdoors)
+"e" = (
+/obj/item/greentext{
+	light_color = "#64C864";
+	light_range = 1
+	},
+/turf/open/floor/mineral/diamond,
+/area/icemoon/surface/outdoors)
+"f" = (
+/obj/structure/falsewall/plastitanium,
+/obj/structure/sign/poster/contraband/lusty_xenomorph,
```

Comment:
put an actual lusty xenomorph maid in here or no balls

Reply:
the joke is that it's the lust ruin but it only has to do with lust as in lusting after an object like the greentext

Reply:
ahhhh